inst/scripts/query-pubmed-grants.R

```diff
@@ -220,11 +225,12 @@ pmids_df <- pmids_df %>% group_by(pmid) %>% reframe(
 # Take only pmids not in the portal already
 pmids_df <-
   pmids_df[!(pmids_df$pmid %in% pubs_exisiting$PubmedId),]
-
+if (nrow(pmids_df) == 0) {
```

Comment:
The code fails if there are 0 rows to update.  We don't have to throw an error, but it was the simple thing to do.